### ISSUE TITLE
Ambiguous line label identifier

### ISSUE THREAD

**Issue by retailcoder** [retailcoder/Rubberduck#408]
Consider this [bad but legal] code:

    Public Sub DoSomething()
    Foo: Bar
    End Sub

    Private Sub Foo()
    End Sub

    Private Sub Bar()
    End Sub

The user's intent may have been to call both `Foo` and `Bar` procedures on a single line, but VBA will parse `Foo:` as a line label, and only `Bar` will be executed.

Rubberduck should warn when it encounters a line label identifier that matches a procedure name that's in-scope.

Possible quick-fix: *Convert line label to procedure call* - turn this:

    Public Sub DoSomething()
    Foo: Bar
    End Sub

Into this:

    Public Sub DoSomething()
        Foo
        Bar
    End Sub

---

Another possible fix would be to introduce a `Call` statement:

    Public Sub DoSomething()
        Call Foo() : Bar
    End Sub

But that would fire up #34, and then applying the *Remove obsolete Call statement* quickfix would result in this code:

    Public Sub DoSomething()
    Foo: Bar
    End Sub

In other words, this inspection solves an issue that an edge-case quickfix of #34 introduces in the user's code.

**Comment by daFreeMan** [comment]
Since `Call()` serves no other valid purpose in VBA, as was [discussed here](http://stackoverflow.com/questions/2573597/should-i-use-call-keyword-in-vb-vba/22325284#22325284), I'd say go with the first option:

    Public Sub DoSomething()
      Foo
      Bar
    End Sub

**Comment by retailcoder** [comment]
@daFreeMan indeed. I was merely documenting an oversight in the implementation of the quickfix for #34 :wink:
